src/Redux/api.tsx

```diff
@@ -1006,10 +1017,19 @@ const routes = {
   getPatientInvestigation: {
     path: "/api/v1/patient/{patient_external_id}/investigation/",
     method: "GET",
+    TRes: Type<PaginatedResponse<Investigation>>(),
   },
   editInvestigation: {
     path: "/api/v1/consultation/{consultation_external_id}/investigation/batchUpdate/",
     method: "PUT",
+    TRes: Type<Record<string, never>>(),
+    TBody: Type<{
+      investigations: {
+        external_id: any;
+        value: any;
+        notes: any;
```

Comment:
replace `any` with proper types. Avoid using any if we know for sure what these types are going to be

Reply:
Done